Makefile

```diff
@@ -73,6 +73,18 @@ PHP_CS_FIXER=php -d zend.enable_gc=0 vendor-bin/owncloud-codestyle/vendor/bin/ph
 PHAN=php -d zend.enable_gc=0 vendor-bin/phan/vendor/bin/phan
 PHPSTAN=php -d zend.enable_gc=0 vendor-bin/phpstan/vendor/bin/phpstan
 
+app_name=$(notdir $(CURDIR))
```

Comment:
this is already at line 47
need to sort out merging in the needed/relevant parts up there